package.json

```diff
@@ -20,6 +20,7 @@
         "build:cms:skippable": "test -f packages/admin/cms-admin/lib/index.d.ts && echo 'Skipping CMS build' || $npm_execpath build:cms",
         "build:storybook": "pnpm recursive --filter '@comet/admin*' run build && pnpm --filter comet-admin-stories run build-storybook",
         "build:lib": "pnpm recursive --filter '@comet/*' run build",
+        "build:docs": "pnpm recursive --filter '@comet/admin*' --filter 'comet-docs' run build",
```

Comment:
Does this ensure that the admin build is executed before the docs build? Maybe `pnpm recursive --filter '@comet/admin*' run build && pnpm --filter 'comet-docs' run build` is safer?

Reply:
> Does this ensure that the admin build is executed before the docs build?

yes, pnpm knows the dependencies and builds in the correct order